<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password_Generator</class>
 <widget class="QMainWindow" name="Password_Generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password_Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/image/img/froggi (2).png</normaloff>:/image/img/froggi (2).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="add">
       <property name="text">
        <string>Add to password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="numbers">
       <property name="text">
        <string>numbers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lowercase_letters">
       <property name="text">
        <string>lowercase letters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="capital_letters">
       <property name="text">
        <string>capital letters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="special_characters">
       <property name="text">
        <string>special characters</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>237</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="display">
       <property name="text">
        <string>Number of password: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="number_of_pass"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>236</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="length">
       <property name="text">
        <string>Length of password:   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="length_of_pass"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="ListPasswords">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
